README.rst

```diff
@@ -59,6 +59,17 @@ Note: The pagination is useful to retrieve a large bunch of resources,
 because a single request may fail (timeout). This is enabled with both
 parameters *allow_pagination* and *pagination_max_limit* as shown above.
 
+
+Configuration of policy.json file
+
+.. code-block:: yaml
+
+    neutron:
+      server:
+        ....
+        policy:
+          create_subnet: 'rule:admin_or_network_owner'
```

Comment:
please update CI, attach updated tests/pillar/* and possibly even tests/integration (verification script)